OptimizelySDK.Travis.sln

```diff
@@ -13,6 +13,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		README.md = README.md
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OptimizelySDK.Net35", "OptimizelySDK.Net35\OptimizelySDK.Net35.csproj", "{C8FF7012-37B7-4D64-AB45-0C62195302EC}"
```

Comment:
What are these changes for?

Reply:
Initially OptimizelySDK.Travis.sln was only using OptimizelySDK\OptimizelySDK.csproj. @msohailhussain added Net35 and Net40 in VS so that those dlls would get generated too. These changes is the result of including those new projects.